wscreen.go

```diff
@@ -19,6 +19,7 @@ package tcell
 
 import (
 	"errors"
```

Comment:
Please move this import into a second stanza -- I prefer to have import stanzas (groups separated by a blank line) as standard library, 3rd party packages, and finally local packages.)

Otherwise this looks good.

Reply:
Actually, I'm just going to fix this because I think another change here is needed.